backend/tests/test_titles_endpoint.py

```diff
@@ -25,7 +25,7 @@ async def test_titles_endpoint_get_title(title_with_data, book_dict):
     assert response.status_code == 200
     assert response.json() == {
         "ident": title_with_data.ident,
-        "languages": [lang.code for lang in title_with_data.languages],
-        "tags": [tag.name for tag in title_with_data.tags],
+        "languages": ["eng"],
```

Comment:
Why are you doing this? We are using a fixture so we don't want to mix fixture stuff with hard coded values.
Also why wouldn't the tags be present?

Reply:
@rgaudin okay, reverted

Reply:
Thanks but you're right ; as I described in my reply to your comment above ; we should compare with book_dict data (or another static fixture) and not from `title_with_data` itself. Thanks for bringing that up

Reply:
@rgaudin to do with I had to update `title_with_data` to have the metadata values same as `book_dict`

Reply:
👍